crates/factor-outbound-networking/src/lib.rs

```diff
@@ -1,8 +1,6 @@
 mod config;
 pub mod runtime_config;
 
-use std::{collections::HashMap, sync::Arc};
```

Comment:
Nit: I don't think it's a hard rule, but we tend to keep `std` uses separate from external crates (which are both separate from uses local to the crate). It's not blocking, but I would consider reverting this.

Reply:
I'm not sure i follow. Are you saying to not use these dependencies, not use them in the root of the crate or change the formatting somehow? Other factors use std libs so i am assuming that is not it: https://github.com/kate-goldenring/spin/blob/6d29d49d5b2d9aad5b13f4c95602a2c4a77b16e8/crates/factor-key-value/src/util.rs#L5-L6

Reply:
Sorry - this is all about formatting of the `use` statements. Typically `use` statements are grouped into three groups:
* `std` lib uses (e.g., std::fmt::Display)
* external crate uses (e.g., `tokio::task::spawn`) 
* uses local to the crate (e.g., `crate::foo`)

These three groups are then separated by an empty line. 

There are a million exceptions, and it's not really important, so I only bring it up since you moved this line for seemingly no other reason than aesthetics. Feel free to ignore my comment 😄